web/modules/custom/joinup_licence/src/Controller/LicenceComparerController.php

```diff
@@ -273,24 +274,23 @@ protected function buildRow(string $category, string $label, array $items): arra
       [
         'data' => $label,
         'class' => [
-          'licence-sidebar',
-          'licence-cell',
+          'licence-comparer__sidebar-cell',
           'licence-filter--' . strtolower($category),
         ],
       ],
     ];
 
     foreach ($items as $enabled) {
       $row[] = [
-        'data' => $enabled ? 'x' : '',
+        'data' => $enabled ? ['#markup' => '<span class="icon icon--check-2"></span>'] : '',
```

Comment:
Instead of adapting the functionality to satisfy the tests, I would suggest to adapt the test coverage. Could be something like "Then I see an orange check in the BSD-2 column".

Reply:
In the tests we can check for the presence of a `span.icon--check-2` element inside the cell to know if the contents match.

Reply:
There is a `BehatHookTableEventDispatcher` which we can probably hook into to manipulate the data. We can use the `@AfterTableFetch` annotation for this. See for an example `FeatureContext::stripScreenReaderElements()` (which is responsible for stripping the `x`'s, but this is wanted behavior ;) ).

Or we can do it by creating a `JlaTableContext` class which extends `RawTableContext` and then do a substitution of the checkmarks in a custom step.

Reply:
Actually, @pfrenssen's suggestion is very good. I've implemented it in d3ba089